src/java.base/share/classes/java/math/RoundingMode.java

```diff
@@ -39,6 +39,21 @@
  * considered as a numerical value, the discarded fraction could have
  * an absolute value greater than one.
  *
+ * <p>More generally, a rounding policy defines a mapping from the
+ * real numbers to a subset of representable values. In the case of
+ * {@link BigDecimal}, the representable values are a function of the
+ * {@linkplain MathContext#getPrecision() precision} being used in the
+ * computation. Assuming the mathematical result is within the
+ * exponent range of {@code BigDecimal}, the mathematical result will
+ * be exactly representable in the result precision or fall between
+ * two representable values. In the case of falling between two
```

Comment:
Perhaps better would be
`two adjacent representable values.`

Reply:
Agreed; adjusted in subsequent push.